README.md

```diff
@@ -81,6 +81,7 @@ configuration file.
 | **_github_light_colorblind_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_light_colorblind.yml)| ![github_light_colorblind](images/github_light_colorblind.png) |
 | **_github_light_high_contrast_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_light_high_contrast.yml)| ![github_light_high_contrast](images/github_light_high_contrast.png)|
 | **_github_light_tritanopia_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_light_tritanopia.yml)| ![github_light_tritanopia](images/github_light_tritanopia.png) |
+|                                                                   **_google_**                                                                    |                ![falcon](images/google.png)                |
```

Comment:
Can you add a source for the theme?

Reply:
I got it from this repo: https://github.com/ianriva/xfce4-terminal-themes 
I can't find any "official" source, so is this source enough?: 
https://github.com/ianriva/xfce4-terminal-themes/blob/master/colorschemes/google.theme

Reply:
I guess we can just keep it like this and if anyone has problems with specific colors they can add a source with their resources instead.